<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userLogin</class>
 <widget class="QWidget" name="userLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>285</width>
     <height>210</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="loginLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Nom d'Utilisateur</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Mot de Passe</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>NIP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pinEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Confirmer NIP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pinConfirmEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
